standard/classes.md

```diff
@@ -2090,6 +2095,20 @@ When a formal parameter is a value parameter, the corresponding argument in a me
 
 A method is permitted to assign new values to a value parameter. Such assignments only affect the local storage location represented by the value parameter—they have no effect on the actual argument given in the method invocation.
 
+#### §method-input-parameters-new-clause Input parameters
```

Comment:
Was this meant to be a section number?

Reply:
It will be. The automatic section renumbering tool runs when we merge a PR into the main branch. It renumbers any sections, and updates all links based on the new numbers. That key lets us add links to new sections in the same PR.